engines/experiment/examples/plugins/hardcoded/Makefile

```diff
@@ -15,4 +17,9 @@ vendor:
 build-image: vendor
 	@$(eval IMAGE_TAG = $(if $(DOCKER_REGISTRY),$(DOCKER_REGISTRY)/,)${PLUGIN_NAME}-engine-plugin:${VERSION})
 	@echo "Building docker image: ${IMAGE_TAG}"
-	docker build --tag ${IMAGE_TAG} .
\ No newline at end of file
+	docker build --tag ${IMAGE_TAG} .
+
+.PHONY: proprietary_exp_plugin
+proprietary_exp_plugin:
```

Comment:
Nit: Can we use hyphen as the separator in the target name, to match others in this file / `api/Makefile` ?